src/Storages/VirtualColumnUtils.h

```diff
@@ -26,9 +26,13 @@ namespace VirtualColumnUtils
 ///
 /// Otherwise calling filter*() outside applyFilters() will throw "Not-ready Set is passed"
 /// if there are subqueries.
+///
+/// Similar to filterBlockWithExpression(buildFilterExpression(splitFilterDagForAllowedInputs(...)))./// Similar to filterBlockWithQuery, but uses ActionsDAG as a predicate.
+/// Basically it is filterBlockWithDAG(splitFilterDagForAllowedInputs).
```

Comment:
Looks a bit messy. Is it after resolving conflicts?

Reply:
Yes, I will have a look at it. At merge time I just kept both text but forgot to check them.